### PR TITLE
Update netsim instructions area to use short instructions

### PR DESCRIPTION
**Pull request by dmcavoy**
The instructions area on NetSim was broken a long time ago when we changed `instructions` to `shortInstructions`. This fixes that issue.

Screen Shot 2020-06-23 at 2 57 31 PM

Slack Thread: https://codedotorg.slack.com/archives/CNZP84FJ5/p1592918229372800

### CODE REVIEW

**Comment by dmcavoy** [comment]
@madelynkasula @clareconstantine We should probably have a test to cover netsim better so that things like this don't break going forward. I don't know if LP or StarLabs owns that. Can one of you take the follow up to add more tests here?

File: apps/src/netsim/netsim.js [hunk 1]
```diff
@@ -219,7 +219,7 @@ NetSim.prototype.init = function(config) {
     return page({
       data: {
         localeDirection: getStore().getState().isRtl ? 'rtl' : 'ltr',
-        instructions: this.level.instructions
+        instructions: this.level.shortInstructions
```
Comment (islemaster) [inline apps/src/netsim/netsim.js:222]: Are shortInstructions guaranteed to be available?  I think in the level I tested this morning there were only longInstructions... but it was one of the /s/netsim levels so maybe that's not typical.

Comment (dmcavoy) [reply apps/src/netsim/netsim.js:222]: I don't think it's guaranteed to be there. Is there something you think we should do if there are no short instructions? Or is it fine to just leave that space blank?

Comment (islemaster) [reply apps/src/netsim/netsim.js:222]: Now that I think about it, we can probably leave the space blank and lean on level designers to make sure short instructions are provided for NetSim levels.

**Review by islemaster — APPROVED** [review]
:tada: Thank you for fixing!

**Comment by maddiedierker** [comment]
thanks, dani! that's *labs land, so i've logged [STAR-1171](https://codedotorg.atlassian.net/browse/STAR-1171) to track

**Review by maddiedierker — APPROVED** [review]
thank you for this!

**Pull request merged**